plugins/event_source/alertmanager.py

```diff
@@ -74,14 +109,18 @@ def clean_host(host):
 async def main(queue: asyncio.Queue, args: Dict[str, Any]):
     app = web.Application()
     app["queue"] = queue
+    app["data_host_path"] = args.get("data_host_path", "labels.instance")
+    app["data_path_separator"] = args.get("data_path_separator", ".")
+    app["data_alerts_path"] = args.get("data_alerts_path", "alerts")
```

Comment:
Type is not enforced. Should try to convert them to string before, otherwise you can have uncontrolled exceptions. 
Only `KeyError` exception is managed.

Reply:
Do you mean `skip_original_data` in next line? I add `bool()` to convert from string.

Reply:
No, I mean `data_host_path`, `data_path_separator` and `data_alerts_path` If those values are not strings, you will have uncontrolled exceptions from `dpath`.  You must convert them to strings before or check if are strings before calling `dpath.util.get`